docs/launching-apps/progress_thread.rst

```diff
@@ -0,0 +1,70 @@
+.. _async-progress-thread-label:
+
+Asynchronous progress thread
+============================
+
+OpenMPI provides an experimental support of software-based asynchronous
```

Comment:
"Open MPI" (with a space, without the quotes, obviously).  Same correction is needed for the other places in this PR that the docs mention "OpenMPI".

Sorry, I know this is a nit, but this is the official Open MPI docs and we need to get the name right.